### PR TITLE
Update translations from Fedora Weblate

### PR DESCRIPTION
**Pull request by cockpituous**
Update translations from Fedora Weblate
 * [x] po-refresh

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -5,20 +5,22 @@
...
+"Last-Translator: Massimiliano Torromeo <massimiliano.torromeo@gmail.com>\n"
+"Language-Team: Italian <https://translate.fedoraproject.org/projects/cockpit/"
+"master/it/>\n"
 "Language: it\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=2; plural=(n != 1)\n"
-"X-Generator: Zanata 4.6.2\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
```
Comment (marusak) [inline po/it.po]: ehm... but I __think__ that in this case it should not be a problem, lets see.

**Comment by martinpitt** [comment]
@marusak: fedora-31 passed, and rhel-* at least passed check-pages.

However, [machines tests fail on rhel-8-2](https://logs.cockpit-project.org/logs/pull-13634-20200229-220608-5504de3c-rhel-8-2/log.html#89-2), same on -distropkg, due to ambiguous CSS selectors. Is that fallout from the non-destructive tests somehow?

**Comment by marusak** [comment]
> However, machines tests fail on rhel-8-2, same on -distropkg, due to ambiguous CSS selectors. Is that fallout from the non-destructive tests somehow?

I cannot tell. Or, I am not even sure what is happening. It is trying to locate VM-name - which would fail exactly in this way if there were more than one VM. But both screenshot and html show only one VM. It might be that it is a timing issue and there are two VMs for a second (one is still being destroyed when this is checked, gets destroyed and then the screenshot is taken).
Two possible solutions spring to mind:
1. Make `ph_only()` not throw when the selector is not there, instead wait for it. ( I think this is not happening - that when it is once ambiguous it immediately fails without waiting for it.)
2. Test should wait at the beginning that there are no VMs before creating a new one.

**Comment by marusak** [comment]
Oh wait, I just realized that it is waiting for 'subVMTest1' but there is 'vmNotInstalled' so this is definitely the case of broken nondestructive case. I'll take a look.

**Comment by marusak** [comment]
The parser is broken: https://github.com/mikeedwards/po2json/issues/87
But I don't know the exact details why it does not work only in some languages. (Seems related to the fact that these languages have only one form for all plurals).

**Pull request merged**